modulus/models/layers/fused_silu.py

```diff
@@ -13,16 +13,22 @@
 # limitations under the License.
 
 import functools
+from typing import Tuple
 
 import torch
 from torch.autograd import Function
 
 try:
-    from nvfuser._C import DataType, Fusion, FusionDefinition
+    import nvfuser
+    from nvfuser import DataType, FusionDefinition
 except ImportError:
-    # accomodating for earlier versions of PyTorch (< 2.0)
-    # which don't need nvfuser as explicit dependency
-    from torch._C._nvfuser import DataType, Fusion, FusionDefinition
+    print(
```

Comment:
Can we use an exception instead of a print statement here?

Reply:
Fixed. Please have a look again